CoreWidgetProvider/Helpers/ChartHelper.cs

```diff
@@ -205,6 +211,25 @@ private static XElement CreateBorderBox(int height, int width)
         return boxElement;
     }
 
+    private static void AddGridLines(XElement chart)
+    {
+        for (var i = 1; i <= NumOfYGridlines; i++)
+        {
+            var lineElement = new XElement(PolylineElement);
+            lineElement.SetAttributeValue(PointsAttr, $"0,{ChartHeight - (i * YGridlineHeight)} {ChartWidth},{ChartHeight - (i * YGridlineHeight)}");
+            lineElement.SetAttributeValue(StyleAttr, LightGrayBoxStyle);
+            chart.Add(lineElement);
+        }
+
+        for (var i = 1; i <= NumOfXGridlines; i++)
+        {
+            var lineElement = new XElement(PolylineElement);
+            lineElement.SetAttributeValue(PointsAttr, $"{i * XGridlineHeight},0 {i * XGridlineHeight},{ChartHeight}");
+            lineElement.SetAttributeValue(StyleAttr, LightGrayBoxStyle);
+            chart.Add(lineElement);
+        }

```

Comment:
This introduces 'static' vertical gridlines to the chart, which is different from the way the charts are presented in Task Manager. If you view the same chart in Task Manager (say, the CPU usage chart), you'll notice that the vertical gridlines _move_ as time progresses. 

In this implementation the vertical gridlines stay in one place, are we alright with that?

Reply:
I suspect we want the gridlines to move in the same way Task Manager does, but I've never asked about it.